Packs/IronPort/Integrations/CiscoEmailSecurityApplianceIronPortV2/README.md

```diff
@@ -1442,3 +1442,148 @@ https://www.cisco.com/c/dam/en/us/td/docs/security/esa/esa14-0/api/AsyncOS-14-0-
 >|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|
 >| 1 | 12 | 1345 | 0 | 1 | 0 | 19 | 15 | 4 | 0 | 0 | 0 | 0 | 0 | 3 | 4 | 0 | 4 | 179 | 5 | 0 | 1567 | 1 | 1383 | 0 | 0 |
 
+
+### cisco-esa-dictionary-add
```

Comment:
Add to the readme commands examples as well and it's outputs.

Reply:
Do you mean human readable output? because there is not context output beside for the cisco-esa-dictionary-list command.